### PR TITLE
Warn user if tipping `http` URL

### LINKED ISSUE #953: Warn if users try to tip `http` URL

### PR DESCRIPTION
**Pull request by imollov**
closes #953

url_status_not_secure

### CODE REVIEW

File: src/assets/notSecureUrl.svg [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<svg width="24" height="24" viewBox="2 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (mradkov) [inline src/assets/notSecureUrl.svg:1]:
```suggestion
<svg viewBox="2 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment (mradkov) [reply src/assets/notSecureUrl.svg:1]: Check [this](https://github.com/aeternity/superhero-ui/pull/897#discussion_r556475843) discussion.

File: src/utils/index.js [hunk 2]
```diff
@@ -61,6 +61,7 @@ export const urlStatus = (tipUrl, verifiedUrls, blacklistedUrls) => {
   const url = twitterProfile || tipUrl;
 
   if (blacklistedUrls.some((u) => url.includes(u))) return 'blacklisted';
+  if (url.startsWith('http') && !url.startsWith('https:')) return 'not-secure';
```
Comment (AtanasKrondev) [inline src/utils/index.js:64]:
```suggestion
  if (!url.startsWith('https:')) return 'not-secure';
```

Shouldn't be enough? 
Looks good in general

**Review by AtanasKrondev — APPROVED** [review]

File: src/utils/index.js (hunk 2 above)
Comment (imollov) [reply src/utils/index.js:64]: It avoids showing the `not-secure` status prematurely to the user when the URL is manually typed.

**Pull request merged**